src/spot_skills/src/spot_skills/spot_ros_wrapper.py

```diff
@@ -67,6 +74,16 @@ def __init__(self) -> None:
         self._arm_action_server.start()
         rospy.loginfo(f"[{self._arm_action_name}] Action server has started.")
 
+        self._gripper_action_name = "end_effector_controller/gripper_action"
```

Comment:
Which name?

Reply:
I was able to call my own server using `"/gripper_controller/gripper_action"`, but it seems like I can pick anything because MoveIt doesn't directly interface with this particular topic.